spec/requests/v1/visitor_can_see_list_of_recipes_spec.rb

```diff
@@ -0,0 +1,45 @@
+RSpec.describe 'GET articles index' do 
+  describe 'Listing articles' do 
+    let!(:recipes) { 2.times { create(:recipe) } }
+    let(:headers) {{ HTTP_ACCEPT: "application/json" }
+    before do
+      get '/v1/recipes'
+      headers: headers
```

Comment:
You need to make the headers part of the get request. Use a comma

Reply:
change articles to recipes :)